docs/extensibility-http.md

```diff
@@ -0,0 +1,525 @@
+
+Then create 9 Services:
+
+```bash
+for NUM in {1..9}
+do
+  # Services are uniquely named
+  # The service uses the Pods port: 8080
+  kubectl expose deployment/device \
+  --name=device-${NUM} \
+  --port=8080 \
+  --target-port=8080 \
+  --labels=project=akri,broker=http,function=device
+done
+```
+
+> Optional: check one the services:
+>
+> ```bash
+> kubectl run curl \
+> --stdin --tty --rm \
+> --image=radial/busyboxplus:curl
+> ```
+>
+> Then, pick a value for `X` between 1 and 9:
+>
+> ```bash
+> X=6
+> curl device-${X}:8080
+> curl device-${X}:8080/
+> curl http://device-${X}:8080/
+> curl http://device-${X}.default:8080/
+> ```
+>
+> Any or all of these should return a (random) 'sensor' value.
+
+Then apply `discovery.yaml` to create a Deployment (called `discovery`) and a Pod (called `discovery-...`):
+
+```bash
+kubectl apply --filename=./discovery.yaml
+```
+
+Then create a Service (called `discovery`) using the deployment:
+
+```bash
+kubectl expose deployment/discovery \
+--name=discovery \
+--port=9999 \
+--target-port=9999 \
+--labels=project=akri,broker=http,function=discovery
+```
+
+> Optional: check the service to confirm that it reports a list of devices correctly using:
+> 
+> ```bash
+> kubectl run curl \
+> --rm --stdin --tty \
+> --image=radial/busyboxplus:curl
+> ```
+>
+> Then, curl the service's endpoint:
+>
+> ```bash
+> curl discovery:9999/
+> ```
+>
+> This should return a list of 9 devices, of the form `http://device-X:8080`
+
+## Deploy Akri
+
+> Optional: If you've previous installed Akri and wish to reset, you may:
+>
+> ```bash
+> # Delete Akri Helm
+> sudo microk8s.helm3 uninstall akri
+>
+> # Delete Akri CRDs
+> kubectl delete crd/configurations.akri.sh
+> kubectl delete crd/instances.akri.sh
+> ```
+
+Deploy the revised (!) Helm Chart to your cluster:
+
+```bash
+HOST="ghcr.io"
+USER="[[GITHUB-USER]]"
+REPO="${HOST}/${USER}"
+VERS="v$(cat version.txt)-amd64"
+
+sudo microk8s.helm3 install akri ./akri/deployment/helm \
+--set imagePullSecrets[0].name="${HOST}" \
+--set agent.image.repository="${REPO}/agent" \
+--set agent.image.tag="${VERS}" \
+--set controller.image.repository="${REPO}/controller" \
+--set controller.image.tag="${VERS}"
+```
+
+> **NOTE** the Akri SemVer (e.g. `0.0.41`) is reflected in `./version.txt` but the tags must be prefixed with `v` and postfixed with the architecture (e.g. `-amd64`)
+
+Check using `kubectl get pods` and look for a pod named `akri-agent-...` and another named `akri-controller...` and that they're both `RUNNING`.
+
+Alternatively, you may:
+
+```bash
+kubectl get pods --selector=name=akri-agent
+kubectl get pods --selector=app=akri-controller
+```
+
+Also:
+
+```bash
+kubectl get crds --output=name
+customresourcedefinition.apiextensions.k8s.io/configurations.akri.sh
+customresourcedefinition.apiextensions.k8s.io/instances.akri.sh
+```
+
+## Deploy standalone Broker
+
+Now we can deploy the (standlone) broker.
```

Comment:
it would be good to have a section that explained why there is a standalone broker AND a grpc broker/client

Reply:
Fixed: https://github.com/deislabs/akri/pull/135/commits/5028a8a8d1f1545d36c38795e6edf7145399827e